Docker/test/selenium/wdio.conf.js

```diff
@@ -49,19 +57,33 @@ exports.config = {
 	// ===================
 
 	// Level of verbosity: "trace", "debug", "info", "warn", "error", "silent"
-	logLevel: process.env.SELENIUM_LOG_LEVEL || 'warn',
+	logLevel: process.env.SELENIUM_LOG_LEVEL || 'error',
```

Comment:
Do we want a --verbose option to go back to 'warn', if not output everything that is now in the logs?

Reply:
That would be nice, but don't want to add the command line arg parsing between the Makefile and Shell Scripts for that (and a couple other options) until things settle a bit more. It would be slightly more than trivial to add that right now, especially given the cost of "testing test runs". All for this being a feature ticket for future :)